node_monitor/telegram_bot.py

```diff
@@ -0,0 +1,25 @@
+import requests
+import logging
+from node_monitor.load_config import telegramBotToken, telegramChatId, telegramChannelId
+
+
+class TelegramBot():
```

Comment:
I like the idea of using class here since it follows the convention we use with DiscordBot and it wraps all the functionality into a logical component. Especially since these functions have side effects.

That being said, this class contains no state. Let's use it as a namespace and mark all the methods within with the `@staticmethod` decorator.


Just FYI Generally classes are used only when you have to encapsulate mutable state. 
I also like using them when my functions have side effects. I like to keep my functions pure and wrap stuff that mutates data in a class. I try and stay away from mutable state.